docs/react-testing-library/faq.md

```diff
@@ -178,6 +178,25 @@ snapshotDiff(firstVersion, container.cloneNode(true))
 
 </details>
 
+<details>
+  
+<summary>How do I fix "an update was not wrapped in act(...)" warnings?</summary>
+
+This warning is usually caused by an async operation causing an update after
+the test has already finished. There are 2 approaches to resolve it:
+
+1. Wait for the result of the operation in your test by using one of
```

Comment:
Wanted to make sure people knew that any of the async utilities will solve this. And in many cases using a `find*` query makes a lot of sense, so I changed the example to that.